CHANGELOG.md

```diff
@@ -86,6 +86,8 @@
 + Parse errors have a new format, similar to GHC 8.2's default, due to a
   switch from Trifecta to Megaparsec for parsing.  This could possibly cause
   some subtle deviations in parsing from previous versions of Idris.
++ Non-parse errors and warnings have mostly been converted to mimic the parse
```

Comment:
That is the most humble description of this update I can think of 😄 .
Maybe consider giving an example, and write that the error messages have been improved and include the affected lines now?